frontend/src/app/components/register/register.component.html

```diff
@@ -1,13 +1,14 @@
-<form (ngSubmit)="processForm()"></form>
+
 <section class="hero is-info">
     <div class="container">
         <div style="text-align: center">
             <h1 class="title">Sign Up!</h1>
-            <input [(ngModel)]="register.username" placeholder="Username" class="input">
+            <form>
+            <input [(ngModel)]="register.username" name="firstname" placeholder="Username" class="input">
```

Comment:
`name="firstname"` but all the other attributes refer to `username` is that on purpose?

Reply:
No that's me being stupid